RESTier.sln

```diff
@@ -45,58 +45,85 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Restier.Providers
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		CodeAnalysis|Any CPU = CodeAnalysis|Any CPU
```

Comment:
Auto added by IDE, I removed once, and then found it is auto added again, so I did not remove them.